truss/contexts/image_builder/serving_image_builder.py

```diff
@@ -200,12 +204,12 @@ def update_model_name(config: TrussConfig, model_key: str) -> str:
     model_name = config.build.arguments[model_key]
     if "gs://" in model_name:
         # if we are pulling from a gs bucket, we want to alias it as a part of the cache
-        model_to_cache = HuggingFaceModel(model_name)
-        config.hf_cache.models.append(model_to_cache)
+        model_to_cache = ModelRepo(model_name)
+        config.model_cache.models.append(model_to_cache)
 
         config.build.arguments[
             model_key
-        ] = f"/app/hf_cache/{model_name.replace('gs://', '')}"
+        ] = f"/app/model_cache/{model_name.replace('gs://', '')}"
```

Comment:
could we move this string into a function w/ comments? it's not clear to me why we need to do this transformation of the config object

Reply:
This is very specifically for TGI/vLLM, where the model maybe specified directly as an HF repo. If it's a GCS or S3 bucket, we want to alias that bucket to the cache and make sure the model server pulls from the cache instead of throwing an error.